python/tvm/relay/ir_pass.py

```diff
@@ -741,6 +741,10 @@ def alter_op_layout(expr):
     return _ir_pass.AlterOpLayout(expr)
 
 
+def alter_operator(expr):
+    return _ir_pass.alter_operator(expr)
```

Comment:
A description would be useful here. I would also be interested in knowing that it differs from Alter_op_layout.